static/app/views/settings/projectDebugFiles/utils.tsx

```diff
@@ -1,17 +1,39 @@
 import {t} from 'sentry/locale';
 import {DebugFile, DebugFileFeature, DebugFileType} from 'sentry/types/debugFiles';
 
-export function getFileType(dsym: DebugFile) {
-  switch (dsym.data?.type) {
-    case DebugFileType.EXE:
-      return t('executable');
-    case DebugFileType.DBG:
-      return t('debug companion');
-    case DebugFileType.LIB:
-      return t('dynamic library');
-    default:
-      return null;
-  }
+const PRETTY_SYMBOL_TYPES = {
+  proguard: t('ProGuard mapping'),
+  breakpad: t('Breadpad'),
+  macho: t('Mach-O'),
+  elf: t('ELF'),
+  pe: t('PE'),
```

Comment:
do we need to translate these files? I am not sure if there is a translation in another language

Reply:
I don’t thing so. The existing code at least applied translations to `"proguard mapping"`, so maybe those are worth translating, others not so much.